### PR TITLE
added Windows 10 UWP implementation for resolving the native library name…

### PR DESCRIPTION
**Pull request by vladkol**
Once built with [vcpkg](https://github.com/Microsoft/vcpkg), assimp can be used with this library. 
They don't use different names for x64, x86 and ARM DLLs, so a proper library must be used during build time. 

P.S. 
Should we merge it to to the original assimp-net repo?

### CODE REVIEW

File: AssimpNet/Unmanaged/AssimpLibrary.cs [hunk 1]
```diff
@@ -1464,6 +1464,9 @@ internal static class AssimpDefaultLibraryPath
         public const String DefaultWindows32BitPath = "Assimp32.dll";
         public const String DefaultWindows64BitPath = "Assimp64.dll";
 
+        public const String DefaultUWP32BitPath = "assimp-vc140-mt.dll";
```
Comment (mellinoe) [inline AssimpNet/Unmanaged/AssimpLibrary.cs:1467]: What's up with these names? Are they not configurable with this vcpkg thing? It's not a problem one way or the other, they just seem odd.

**Comment by mellinoe** [comment]
This looks like the right way to do it to me. Thanks!

> Should we merge it to to the original assimp-net repo?

Yeah, I'd like to. I don't think anyone is actively maintaining it, though. Nothing has been merged for a couple of years.

**Comment by vladkol** [comment]
This is the default naming of assimp dll as it comes from the build. They define it [here](https://github.com/assimp/assimp/blob/12908506abd5a5f47e14a493448ec16652ecd38e/CMakeLists.txt) as assimp-${ASSIMP_MSVC_VERSION}-mt.dll

**Comment by mellinoe** [comment]
@vladkol I see -- you are right about that. It's not limited to UWP, though. All Windows versions should have that name. However, this repo actually already includes pre-built binaries for 32- and 64-bit Windows, but with different names (Assimp32/Assim64). Maybe we should rename the UWP binaries and put them in the repo as well?

**Comment by vladkol** [comment]
>  don't think anyone is actively maintaining it, though 
It seems they moved to bitbucket: https://bitbucket.org/Starnick/assimpnet At least, this is where there are more updates, NuGet package home page is there, and other projects the original maintainers are working on.

**Comment by vladkol** [comment]
> Maybe we should rename the UWP binaries and put them in the repo as well? 
Seems reasonable. Let me send the binaries over.

**Comment by mellinoe** [comment]
Can you just include the binaries in the PR? I think they can just be in a "UWP" folder next to the existing ones.

**Comment by vladkol** [comment]
done

**Comment by mellinoe** [comment]
Could you remove the PDB's and all of the debug versions? Probably best to just have the release versions for each arch.

**Comment by mellinoe** [comment]
Looks okay now. Thanks!

**Pull request merged**